src/utils/alert.ts

```diff
@@ -72,34 +72,62 @@ export const validateResponse = (
   return checks
 }
 
-export const getMessageForAlert = (
-  alert: string,
-  url: string,
-  ipAddress: string,
+export const getMessageForAlert = ({
+  alert,
+  url,
+  ipAddress,
+  status,
+  trueThreshold,
+}: {
+  alert: string
+  url: string
+  ipAddress: string
   status: string
-): {
+  trueThreshold: number
```

Comment:
can we change this variable name?
`trueThreshold` is too ambiguous.

Reply:
if this is changed, don't forget to change in the other files. 

How about:

- `incidentThreshold` instead of trueThreshold
- `recoveryThreshold` instead of falseThreshold

Reply:
fixed